01/app.js

```diff
@@ -1 +1,12 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM')
+
+const newestItem = document.querySelector('.comments__item, .comments__item--newest')
+
+if (newestItem !== null) {
+	const dataEl = newestItem.querySelectorAll('[data-info]')
+	if (dataEl) {
```

Comment:
W tym przypadku nie trzeba robic `if` ponieważ `.querySelectoAll` zawsze zwraca tablicę. Tablica zawsze jest prawdą (nawet pusta). Dlatego jak chcesz mieć warunek to taki: `if(dataEl.length > 0)` lub krócej `if(dataEl.length)`